packages/gatsby/src/db/nodes.d.ts

```diff
@@ -0,0 +1,11 @@
+interface NodeStore {
+  getNodes: () => Array<any>
+  getNode: (id: string) => any | undefined
+  getNodesByType: (type: string) => Array<any>
+  getTypes: () => Array<string>
```

Comment:
```suggestion
  getTypes: () => string[];
```

Reply:
I think it's a good idea to expand the TS linting rules. I'll see about setting this up